src/internal/account.ts

```diff
@@ -170,12 +216,43 @@ export async function getResources(args: {
     path: `accounts/${AccountAddress.from(accountAddress).toString()}/resources`,
     params: {
       ledger_version: options?.ledgerVersion,
-      offset: options?.offset,
-      limit: options?.limit ?? 999,
+      limit: options?.limit ?? 1000,
```

Comment:
was there a reason for it to be 999?

Reply:
idk, but I wanted to make it consistent.

Can change back to 999